budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/pages/user/selectproject/SelectProjectPage.java

```diff
@@ -50,9 +50,14 @@ public SelectProjectPage(Class<? extends WebPage> backlinkPage, PageParameters b
         Form<String> form = new Form<String>(id, new Model<String>("")) {
             @Override
             protected void onSubmit() {
-                ProjectBaseData project = projectService.createProject(getModelObject(), BudgeteerSession.get().getLoggedInUser().getId());
-                BudgeteerSession.get().setProjectId(project.getId());
-                setResponsePage(DashboardPage.class);
+                try {
+                    ProjectBaseData project = projectService.createProject(getModelObject(), BudgeteerSession.get().getLoggedInUser().getId());
+                    BudgeteerSession.get().setProjectId(project.getId());
+                    setResponsePage(DashboardPage.class);
+                }catch (Exception e){
+                    this.error("Project name already in use");
```

Comment:
Use a .properties file for localized strings

Reply:
Changed in new PR